hw/dv/tools/xcelium/xcelium.ccf

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//only collect toggle coverage on dut
+deselect_coverage -toggle -instance tb.dut.*...
+
+// only collect toggle coverage on ports
+set_toggle_portsonly
+// enable toggle scoring of structs and multidim arrays + MDA structs
+set_toggle_scoring -sv_mda -sv_mda_of_struct
+// filter glitches
+set_toggle_strobe 0ps
+//  Filter unreachable/statically constant blocks
+set_com -log
```

Comment:
nit: add newline

Reply:
fixed

Reply:
Newline still appears to be needed (the red symbol is still there).

Reply:
hmm never seen this before - works in local flow.
anyway I added the newline